R/structure.info.R

```diff
@@ -55,4 +69,3 @@ are_adjacent <- function(graph, v1, v2) {
   )
 }
 #' @export are.connected
```

Comment:
Can we also remove this `#' @export` line?

Reply:
sort of embarrassing that this was an "off by one" error :zany_face: thanks for catching it @krlmlr

Reply:
One of the two hard things in programming 🤣